tests/tck/features/VarLengthAcceptance.feature

```diff
@@ -292,6 +296,7 @@ Feature: VarLengthAcceptance
       MATCH (a:A)
       MATCH (a)-[:LIKES*..1]->(c)
       RETURN c.name
+      ORDER BY c.name
```

Comment:
I discussed this a bit with the TCK team:
https://github.com/opencypher/openCypher/issues/370

In short, the acceptance criteria `Then the result should be:` does not guarantee a result order, and we should change `Steps.py` accordingly. Hopefully they'll improve the naming conventions to be more explicit soon.